node-graph/gcore/src/raster/adjustments.rs

```diff
@@ -885,3 +885,12 @@ mod index_node {
 		}
 	}
 }
+
+#[derive(Debug, Clone, Copy)]
+pub struct ImageColorPaletteNode {}
+
+#[node_macro::node_fn(ImageColorPaletteNode)]
+fn image_color_palette(color: Color) -> Color {
```

Comment:
It doesn't really make sense to find the colour palette of just an individual pixel @henrybarreto, you need an image. Also I assume the colour palette generated would need to be several colours i.e. `Vec<Color>`. Perhaps it would be better to instead have this function signature `|image: ImageFrame<Color>| -> Vec<Color>`.

The entry in the node_registry would also have to be updated to be `register_node!(graphene_core::raster::ImageColorPaletteNode, input: ImageFrame<Color>, params: [])`.

Reply:
Nice! Thanks for the tips.